dist/optimized/android/description.svg

```diff
@@ -0,0 +1 @@
+<svg width="28" height="28" viewBox="0 0 28 28" xmlns="http://www.w3.org/2000/svg"><title>description</title><path d="M17 19c0 .55-.45 1-1 1H7c-.55 0-1-.45-1-1s.45-1 1-1h9c.55 0 1 .45 1 1zm5-10c0 .55-.45 1-1 1H7c-.55 0-1-.45-1-1s.45-1 1-1h14c.55 0 1 .45 1 1zm0 5c0 .55-.45 1-1 1H7c-.55 0-1-.45-1-1s.45-1 1-1h14c.55 0 1 .45 1 1zm2 8c0 1.1-.9 2-2 2H6c-1.1 0-2-.9-2-2V6c0-1.1.9-2 2-2h16c1.1 0 2 .9 2 2v16zM22 2H6C3.8 2 2 3.8 2 6v16c0 2.2 1.8 4 4 4h16c2.2 0 4-1.8 4-4V6c0-2.2-1.8-4-4-4z" fill-rule="evenodd"/></svg>
```

Comment:
looks like these files didn't get deleted after you renamed the source to `textblock`

Reply:
good catch, I'll clean